src/akkudoktoreos/prediction/load.py

```diff
@@ -0,0 +1,61 @@
+"""Load forecast module for load predictions."""
+
+from typing import Optional, Set
+
+from pydantic import Field, computed_field
+
+from akkudoktoreos.config.configabc import SettingsBaseModel
+from akkudoktoreos.utils.logutil import get_logger
+
+logger = get_logger(__name__)
+
+
+class LoadCommonSettings(SettingsBaseModel):
+    # Load 0
+    load0_provider: Optional[str] = Field(
```

Comment:
> can we not make this a list of providers?

This would introduce extra logic to read the values from environment variables. Currently I like to stick with this simple scheme.